mooncake-transfer-engine/include/transport/coro_rpc_connector/cororpc_interface.h

```diff
@@ -0,0 +1,83 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include <vector>
+#include <pybind11/pybind11.h>
+
+namespace mooncake {
+
+struct Config;
+
+class CoroRPCInterface {
+   public:
+    struct ReceivedData {
+        std::string source_address;
+        std::string data;
+        size_t data_size = 0;
+
+        pybind11::bytes getBytes() const { return pybind11::bytes(data); }
```

Comment:
zero copy to optimize;
ReceivedData -> shared_ptr:

  py::class_<ReceivedData, std::shared_ptr<ReceivedData>>(m, "Holder")
      .def(py::init<std::string>())
      .def("str_view", &string_holder::str_view);